client/src/components/NavFooter.js

```diff
@@ -0,0 +1,70 @@
+import styled from 'styled-components/macro';
+import { NavLink } from 'react-router-dom';
+
+import HomeButton from '../images/navHome.svg';
+import HomeButtonActive from '../images/navHomeActive.svg';
+import ShelfButton from '../images/navShelves.svg';
+import ShelfButtonActive from '../images/navShelvesActive.svg';
+import BooksButton from '../images/navBooks.svg';
+import BooksButtonActive from '../images/navBooksActive.svg';
+
+export default function NavFooter({ isStatic }) {
+  return (
+    <NavWrapper isStatic={isStatic}>
+      <NavLink to="/home" activeClassName="active">
```

Comment:
As far as I know, the default `activeClassName` is already `active`, so you don't need to set this prop.